free-1.md

```diff
@@ -9,34 +10,40 @@ good way to meet their needs.
 _What need is met by your idea? Who are you helping? What is that person's
 experience like now? What would their experience be like if you could help 
 them?_
+The need is to be able to create calendar events by just typing in information as you would tell it to a peer. (eg. "i have a doctors appointment at 415 pm this thursday at student health services") Without this a person would have to fill in all those fields (what, when, where) individually. If i could help them, talking to a computer would be just as easy as talking to a human.
 
 
 ### Why a language?
 _Why is a DSL appropriate for your user(s)? How does it address the need?_
-
+A language because humans like to talk in languages that are easy to understand and english is at the top of that list for my potential users. 
 
 ### Why you?
 _What excites you about this idea? How did you come up with it?_
+Im excited about this idea because I want to use it! I came up with it when I was talking to Siri and doing this, but I dont think that there is a written equivalent baked into OSX.
 
 
 ### Domain
 _Describe the project's domain in five words._
+People that use computer calendars.
 
 
 ### Interface (syntax)
 _How might the user interact with the language? What does programming look 
 like? Why is this the right way to interact with the problem domain?_ 
+The user will interact with the language by writing in english, programming looks like plain english, and this is the right way to do it because thats the whole point! 
 
 
 ### Operation (semantics)
 _What might happen when a program runs? How does a program interact with the
 user? What kinds of errors might occur, and how might they be communicated to
 the user?_
+When a program runs it will create and save an event in your calendar with the specified parameters, a program would interact with a user by saving the event and potentially giving popup messages to the user if there is an error. There might be an error if the parser misunderstands the text.
 
 
 ### Expressiveness
 _What should be easy to do in this language? What should be possible, but
 difficult? What should be impossible or very difficult?_
+It would be really easy to make calendar events but impossible to do anything else.
```

Comment:
Easy making events how? Your language describes a new way to write events, so describe the expressiveness of writing them.